core/embed/rust/src/ui/model_tr/layout.rs

```diff
@@ -748,6 +748,37 @@ extern "C" fn tutorial(n_args: usize, args: *const Obj, kwargs: *mut Map) -> Obj
     unsafe { util::try_with_args_and_kwargs(n_args, args, kwargs, block) }
 }
 
+extern "C" fn new_centered_popup_middle_button(
```

Comment:
shouldn't this be `show_error` or whichever the equivalent on TT side is?

(in general please try to use semantic names. when we change the layout of "show error", we don't want to be renaming this to `leftaligned_popup_with_right_icon`)

Reply:
Right, renamed to `show_error` in [f813823](https://github.com/trezor/trezor-firmware/pull/3096/commits/f813823b3cfaaf6c6425e630335d01f32267b808)

I am implementing very similar functionality on other current PRs, so after all is merged, we might want to do some review of the whole Rust API